src/reporter/format-utils.xsl

```diff
@@ -119,16 +119,20 @@
   </xsl:for-each>
 
   <!-- Output attributes while performing comparison -->
-  <xsl:for-each select="@*">
+  <xsl:for-each select="attribute()">
+    <xsl:variable name="attribute-to-compare-with" as="attribute()?"
+      select="$node-to-compare-with/attribute()[node-name(.) eq node-name(current())]" />
+
+    <xsl:variable name="display-value" as="xs:string"
+      select="replace(replace(., '&quot;', '&amp;quot;'), '\s(\s+)', '&#x0A;$1')" />

```

Comment:
Did you consider including the surrounding quotation marks in `display-value` (and removing them from the `xsl:text` instructions on lines 143 and 154)? That way, even if the attribute value is empty, the quotation marks provide something to color and draw the user's attention. Oxygen's Compare Files tool includes quotation marks in its attribute value highlighting.

Reply:
Sounds reasonable.
Pushed the suggested change ([result](https://htmlpreview.github.io/?https://github.com/xspec/xspec/blob/292979b0c2f2b425c040655f45a83226571f51f7/test/end-to-end/cases/expected/stylesheet/xspec-serialize-result.html#ELEM-692)) and updated the initial comment (https://github.com/xspec/xspec/pull/704#issue-335147787).